src/components/List/List.js

```diff
@@ -39,12 +112,18 @@ List.propTypes = {
         return error;
     },
     /** Adds dividing lines between the list items */
-    isDivided: PropTypes.bool
+    isDivided: PropTypes.bool,
+    /** onNavigate function callback */
```

Comment:
maybe add to the comments what arguments will be passed to this callback. As well as to the next one.